CHANGELOG.md

```diff
@@ -22,7 +22,8 @@ The types of changes are:
 
 ### Changed
 * Refactor privacy center to be more modular [#1363](https://github.com/ethyca/fidesops/pull/1363)
- * Access support for Square [#1378](https://github.com/ethyca/fidesops/pull/1378)
+* Access support for Square [#1378](https://github.com/ethyca/fidesops/pull/1378)
```

Comment:
The change log message for new connectors should go under the `Added` section unless we're making a fix to an existing connector.